src/components/Articles/ArticleList.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from "react";
+import ArticleCard from "./ArticleCard";
+import axios from "axios";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    axios
+      .get("https://back-end-nna.onrender.com/api/articles")
+      .then((response) => {
+        setArticles(response.data.articles);
```

Comment:
It seems like your response here is not data.articles but just data which gives back the array of articles. 

This means your backend api isnt sending this on the correct key but just to get this working on the frontend for now I would recommend removing the .articles. Once the backend has been fixed for this it can be put back in here as well.